pipeline/backend/kubernetes/kubernetes.go

```diff
@@ -288,13 +281,11 @@ func (e *kube) WaitStep(ctx context.Context, step *types.Step, taskUUID string)
 		return nil, fmt.Errorf("no terminated state found for container %s/%s", pod.Name, cs.Name)
 	}
 
-	bs := &types.State{
+	return &types.State{
 		ExitCode:  int(cs.State.Terminated.ExitCode),
 		Exited:    true,
 		OOMKilled: false,
-	}
-
-	return bs, nil
+	}, nil
```

Comment:
I think this is a slight improvement to the code, however this is not necessary either. If we try to add this change upstream someone from the Woodpecker team might complain that it's unnecessary. I would remove it.